src/redux/books/booksSlice.js

```diff
@@ -0,0 +1,23 @@
+import { createSlice } from '@reduxjs/toolkit';
+
+const initialState = {
+  books: [],
+  isLoading: true,
+};
+
+const booksSlice = createSlice({
+  name: 'books',
+  initialState,
+  reducers: {
+    addBook: (state, action) => {
+      const newBook = action.payload;
+      state.books = state.books.append(newBook);
+    },
+    deleteBook: (state, action) => {
+      const bookId = action.payload.id;
+      state.books = state.books.filter((item) => item.id !== bookId);
+    },
+  },
+});
+
+export default booksSlice.reducer;
```

Comment:
- Nice job exporting bookSlice as reducer. I think we are equally supposed to export addBook and deleteBook as actions. **Hints** `export const { addBook, deleteBook } = bookSlice.actions` Let's kindly export them before merging our PR :smile:

Reply:
Thank you @FrederickMih